web/ui/react-app/src/pages/index.ts

```diff
@@ -7,6 +7,27 @@ import Status from './status/Status';
 import Targets from './targets/Targets';
 import PanelList from './graph/PanelList';
 import TSDBStatus from './tsdbStatus/TSDBStatus';
-import Starting from './starting/Starting';
+import { withStartingIndicator } from '../components/withStartingIndicator';
 
-export { Alerts, Config, Flags, Rules, ServiceDiscovery, Status, Targets, TSDBStatus, PanelList, Starting };
+const AlertsPage = withStartingIndicator(Alerts);
+const ConfigPage = withStartingIndicator(Config);
+const FlagsPage = withStartingIndicator(Flags);
+const RulesPage = withStartingIndicator(Rules);
+const ServiceDiscoveryPage = withStartingIndicator(ServiceDiscovery);
+const StatusPage = withStartingIndicator(Status);
+const TSDBStatusPage = withStartingIndicator(TSDBStatus);
+const TargetsPage = withStartingIndicator(Targets);
+const PanelListPage = withStartingIndicator(PanelList);
+
+// prettier-ignore
```

Comment:
Why's this needed vs. just letting prettier do its thing?

Reply:
Prettier wants a trailing comma, but when I add one, then eslint errors saying there shouldn't be one.

Reply:
It's not complaining about the trailing comma in the imports in `App.tsx` , just the one here.